src/components/AdContainer/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import settings from 'settings';
+import getClassNames from 'helpers/getClassNames';
+import styles from './ad-container.scss';
+
+const AdContainer = ({currentCity = {}, type}) => (
+  <div
+    id={`ad-slot--${type}`}
+    className={getClassNames(styles, ['root', type])}
```

Comment:
There should be some static CSS class in place (not CSS modules based) so that external scripts that integrate the advertising functionality can have a simple simple way of selecting all ad slot placeholders: e.g. `document.getElementsByClassName('ad-slot')`. Very similar to what you did one line above with the "id-attribute".

The current CSS classes do not allow this:

![](https://www.dropbox.com/s/fgjthij2w5wjw9t/Screenshot%202016-10-24%2014.31.06.png?dl=1)

So this line could just be something like:

```
className={`ad-slot ad-slot--${type}`}
```